src/rpc/chain_api.rs

```diff
@@ -320,6 +320,12 @@ pub(crate) async fn chain_get_min_base_fee<DB: Blockstore>(
     Ok(min_base_fee.atto().to_string())
 }
 
+pub(crate) async fn chain_notify<DB: Blockstore>(
+    _data: Data<RPCState<DB>>,
+) -> Result<(), JsonRpcError> {
+    Err(JsonRpcError::METHOD_NOT_FOUND)
+}
+
```

Comment:
Why? Shouldn't the server crate do this automatically?

Reply:
The `ChainNotify` can be removed, yes. It will be implemented in a subsequent PR.